src/plugins/intel_npu/src/plugin/src/plugin.cpp

```diff
@@ -159,6 +149,15 @@ static Config merge_configs(const Config& globalConfig,
                             const std::map<std::string, std::string>& rawConfig,
                             OptionMode mode = OptionMode::Both) {
     Config localConfig = globalConfig;
+    auto it = rawConfig.find(std::string(LOG_LEVEL::key()));
+    
+    if (it != rawConfig.end()) {
```

Comment:
I think this code snippet is repeated compared the one in `void Plugin::set_property`. Is there a way to unify them into one function?

Reply:
Call `update_log_level` function in `merge_configs` and `Plugin::set_property` to update the global log. The implementation of `update_log_level`  function is [here](https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_npu/src/plugin/src/plugin.cpp#L144).